positron/modules/buffer.js

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
```

Comment:
> Nit: I'm worried about these being imported everywhere they are not used (it adds unnecessary cognitive overhead). Also, if we start linting this code, the linter will get upset that things are being defined by never used.

I agree that it's verbose, but I would fix that by injecting those globals into all native binding modules in lieu of simply providing Components, since almost all of those modules access some of those globals (or will, once implemented, for the ones that are currently stubs). I've filed #44 on that.